network-api/networkapi/wagtailpages/templates/wagtailpages/tags/picture_ratios.html

```diff
@@ -0,0 +1,21 @@
+{% load wagtailimages_tags %}
+
+
+<picture class="cms-image {{class_names}}">
+
```

Comment:
Since the alt text is the same for all these images, I would recommend a small refactor:
```
{% with alt_text = unoptimized_image.alt %}
<picture class="cms-image {{class_names}}">

  {% image unoptimized_image fill-.... %}
  {% image unoptimized_image fill-.... %}
  <source ...  alt="{{ alt_text }}">

  {% image unoptimized_image fill-.... %}
  {% image unoptimized_image fill-.... %}
  <source ...  alt="{{ alt_text }}">

  {% image unoptimized_image fill-.... %}
  {% image unoptimized_image fill-.... %}
  <source ...  alt="{{ alt_text }}">

</picture>
{% endwith %}
```

Reply:
done.